plugins/woocommerce/client/blocks/assets/js/blocks/cart/inner-blocks/cart-order-summary-discount/block.tsx

```diff
@@ -23,9 +23,15 @@ const DiscountSlotFill = (): JSX.Element => {
 	return <ExperimentalDiscountsMeta.Slot { ...discountsSlotFillProps } />;
 };
 
-const Block = ( { className }: { className: string } ): JSX.Element => {
+const Block = ( { className }: { className: string } ) => {
 	const { cartTotals, cartCoupons } = useStoreCart();
 	const { removeCoupon, isRemovingCoupon } = useStoreCartCoupons( 'wc/cart' );
+
+	// Hide if there are no coupons to show.
+	if ( ! cartCoupons.length ) {
+		return null;
+	}
```

Comment:
This is hiding the slot/fill - will raise a PR to fix

Reply:
I checked, subscriptions uses `ExperimentalOrderShippingPackages`  and `ExperimentalOrderMeta` the one I am proposing to return is `ExperimentalDiscountsMeta` - a plugin should not display shipping-related UI in this area.

Also, even if plugins _did_ show a form here, it would render anyway if the shopper added a coupon.